packages/containers/src/List/List.test.js

```diff
@@ -99,8 +102,11 @@ describe('Container List', () => {
 		expect(typeof props.list.titleProps.onEditCancel).toBe('function');
 		expect(props.toolbar.filter.placeholder).toBe('find an object');
 		expect(typeof props.toolbar.filter.onFilter).toBe('function');
+		expect(typeof props.toolbar.filter.onToggleFilter).toBe('function');
```

Comment:
I don't think those checks need to be made here. It seems to be for validating the default props, when no one are given.
Here the only test I can see, for the adding expectations and some which where already there before is : verifying the above setting object 'settings' in this file which doesn't test something of the component :/ . I could swap the component rendered with whatever other component and those expectations are still true ^^.

Reply:
added unit tests for those callbacks separately